### PR TITLE
Fix compatibility with navicat

### PR DESCRIPTION
**Pull request by shenli**
We should set all fields to greater than zero. Or navicat will display nothing.

### CODE REVIEW

File: plan/logical_plan_test.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ import (
 var _ = Suite(&testPlanSuite{})
 
 func TestT(t *testing.T) {
-	CustomVerboseFlag = true
+	// CustomVerboseFlag = true
```
Comment (hanfei1991) [inline plan/logical_plan_test.go]: Why comment this?

Comment (shenli) [inline plan/logical_plan_test.go]: It is a mistake.

**Comment by shenli** [comment]
@hanfei1991 @coocood @zimulala  PTAL

File: plan/logical_plan_test.go (hunk 1 above)
Comment (hanfei1991) [inline plan/logical_plan_test.go]: So fix it ?

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**